jsettlers.main.swing/config.prp

```diff
@@ -4,4 +4,4 @@
 resources-folder=../jsettlers.common/resources/
 
 #Folder containing the folders of the graphics and the sound files. If you want to specify multiple possible folders, use a ";" character for separation.
-settlers-folder=/home/michael/.jsettlers;D:/Games/Siedler3;C:/Program Files/siedler 3
\ No newline at end of file
+settlers-folder=/home/david/Development/jsettler_res/

```

Comment:
As mentioned above, it would be easier if this file was autogenerated and not tracked. Or I suppose there's always the local option:
git update-index --assume-unchanged jsettlers.main.swing/config.prp